sorc/link_fv3gfs.sh

```diff
@@ -38,6 +38,8 @@ elif [ $machine = "theia" ]; then
     FIX_DIR="/scratch4/NCEPDEV/global/save/glopara/git/fv3gfs/fix"
 elif [ $machine = "hera" ]; then
     FIX_DIR="/scratch1/NCEPDEV/global/glopara/fix"
+elif [ $machine = "wcoss2" ]; then
+    FIX_DIR="/lfs/h2/emc/global/noscrub/Kate.Friedman/glopara/FIX/fix_nco_gfsv15"
```

Comment:
@KateFriedman-NOAA is this the final resting place for GFS v15 fix data?

Reply:
@WalterKolczynski-NOAA No, I need the global group account space to host these. That will be the final resting place, my copies are meant to be temporary. @arunchawla-NOAA Any update on setting up the global group account on WCOSS2? We have a bunch of stuff to host under it, like fix files for hand-off.

Reply:
Thanks @WalterKolczynski-NOAA and @KateFriedman-NOAA I am also waiting for the final resting place to freeze the code GEFSv12